data_steward/gcloud/bq/__init__.py

```diff
@@ -27,6 +30,8 @@
 trace.set_tracer_provider(tracer_provider)
 tracer = trace.get_tracer(__name__)
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
This will cause issues once deployed to prod. as this tries to overwrite what trace logs to stack driver. I would say please remove the LOGGER.

Reply:
I think this happens only with pipeline_logging. I can still remove it.

Reply:
@nishanthpp93 can you try deploying in test and verify it works, I remember it failing to start the service in app_engine.

Reply:
Removed it.